cypress/support/commands.js

```diff
@@ -97,6 +76,59 @@ Cypress.Commands.add('visitLemma', {prevSubject: false}, (lemmaText, queryParams
   )
 })
 
+/**
+ * Reads the .cypress-user.json file. This file should contains log-in credentials for an admin user.
+ *
+ * Promises an object with {username, password} properties, i.e.,
+ *
+ *    cy.readCypressUserJSON()
+ *      .then(({username, password}) => {
+ *        // use username and password
+ *      })
+ *
+ */
+Cypress.Commands.add('readCypressUserJSON', () => {
+  cy.readFile(CYPRESS_USER_JSON)
+})
```

Comment:
Arrow function with `{}` but no `return`: intentional?

Reply:
Wait... I literally do not know why this works. I added the `return` but it seems to not matter...? This can be chained off of _without_ the `return` statement. `cy` must do some weird magic.